etc/kratos/values.yaml

```diff
@@ -0,0 +1,275 @@
+      {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "$id": "default",
+        "title": "user",
+        "type": "object",
+        "properties": {
+          "traits": {
+            "type": "object",
+            "properties": {
+              "email": {
+                "type": "string",
+                "format": "email",
+                "title": "E-Mail",
+                "minLength": 5,
+                "ory.sh/kratos": {
+                  "credentials": {
+                    "password": {
+                      "identifier": true
+                    },
+                    "totp": {
+                      "account_name": true
+                    }
+                  },
+                  "verification": {
+                    "via": "email"
+                  },
+                  "recovery": {
+                    "via": "email"
+                  }
+                }
+              }
+            },
+            "required": [ "email" ]
+          }
+        },
+        "additionalProperties": false
+      }
+
+  # -- You can customize the emails Kratos is sending (also uncomment config.courier.template_override_path below)
+  emailTemplates: { }
+  # emailTemplates:
+  #   recovery:
+  #     valid:
+  #       subject: Recover access to your account
+  #       body: |-
+  #         Hi, please recover access to your account by clicking the following link:
+  #         <a href="{{ .RecoveryURL }}">{{ .RecoveryURL }}</a>
+  #       plainBody: |-
+  #         Hi, please recover access to your account by clicking the following link: {{ .RecoveryURL }}
+  #     invalid:
+  #       subject: Account access attempted
+  #       body: |-
+  #         Hi, you (or someone else) entered this email address when trying to recover access to an account.
+  #         However, this email address is not on our database of registered users and therefore the attempt has failed. If this was you, check if you signed up using a different address. If this was not you, please ignore this email.
+  #       plainBody: |-
+  #         Hi, you (or someone else) entered this email address when trying to recover access to an account.
+  #   verification:
+  #     valid:
+  #       subject: Please verify your email address
+  #       body: |-
+  #         Hi, please verify your account by clicking the following link:
+  #         <a href="{{ .VerificationURL }}">{{ .VerificationURL }}</a>
+  #       plainBody: |-
+  #         Hi, please verify your account by clicking the following link: {{ .VerificationURL }}
+  #     invalid:
+  #       subject:
+  #       body:
+  #       plainBody:
+
+  config:
+
+    session:
+      # Defines how long a session is active. Once that lifespan has been reached, the user needs to sign in again.
+      lifespan: 24h
+
+      cookie:
+        ##-- If false, cookie is removed when the browser is closed --##
+        persistent: false
+
+    courier:
+      smtp:
+        from_address: radar@thehyve.nl
```

Comment:
But it is configurable, right?

Reply:
It is but I think it should be in `base.yaml` since it will be changed in every installation.

Reply:
agreed

Reply:
Ok the most simple solution would be to write this entry in base.yaml like so:
```
kratos:
  _install: true
  _chart_version: 0.43.0
  _extra_timeout: 0
  ....
  kratos:
    courier:
      smtp:
        from_address: radar@thehyve.nl
```

However, since SMTP settings are also used by MP (and possibly other services), we should consider defining this as global properties and inject this in the `set:` section of helmfile where needed.